internal/bminventory/inventory.go

```diff
@@ -280,7 +313,16 @@ func (b *bareMetalInventory) formatIgnitionFile(cluster *common.Cluster, params
 	if err = tmpl.Execute(buf, ignitionParams); err != nil {
 		return "", err
 	}
-	return buf.String(), nil
+
+	res := buf.String()
+	if cluster.IgnitionConfigOverrides != "" {
+		res, err = mergeIgnitionConfig(buf.Bytes(), []byte(cluster.IgnitionConfigOverrides))
```

Comment:
Add logging.
Also consider logging the full ignition we will use (not here but where we pass it to the create ISO job), so in case the node won't register we could see the ignition it got.

Reply:
I think this is called right before the upload. If we want to log only if the ignition is customized I think this is really the only place to do it.